app/bundles/PointBundle/Tests/Unit/Model/TriggerModelTest.php

```diff
@@ -124,35 +122,9 @@ public function testTriggerEvent(): void
             ->method('find')
             ->willReturn($triggerEvent);
 
-        $this->dispatcher->expects($this->exactly(2))
+        $this->dispatcher->expects($this->exactly(1))
             ->method('dispatch')
             ->withConsecutive(
-                [
-                    $this->callback(
-                        // Emulate a subscriber:
-                        function (TriggerBuilderEvent $event) {
```

Comment:
I know this was this complicated due to Swiftmailer. It should be easier with Symfony Mailer. But does it have to be removed without a replacement?

Reply:
You are right, I forgot to fix this test properly